app/helpers/users_helper.rb

```diff
@@ -0,0 +1,92 @@
+module UsersHelper
+  #
+  # Use this to wrap view elements that the user can't access.
+  # !! Note: this is an *interface*, not *security* feature !!
+  # You need to do all access control at the controller level.
+  #
+  # Example:
+  # <%= if_authorized?(:index,   User)  do link_to('List all users', users_path) end %> |
+  # <%= if_authorized?(:edit,    @user) do link_to('Edit this user', edit_user_path) end %> |
+  # <%= if_authorized?(:destroy, @user) do link_to 'Destroy', @user, :confirm => 'Are you sure?', :method => :delete end %>
+  #
+  #
+  def if_authorized?(action, resource)
+    if authorized?(action, resource)
```

Comment:
Haven't checked this but pretty sure this syntax works right? Codeclimate always asks to fix conditionals like this
```suggestion
    yield(action, resource) if authorized?(action, resource)
```